### PR TITLE
Ruby Account Secret Management Code Snippets

### PR DESCRIPTION
**Pull request by benhayehudi**
## Description

This contains the YAML files for the remaining Ruby Account Secrets management code snippets.

## Deploy Notes

This PR is dependent on the following submodule PR, which needs to be merged first: https://github.com/Nexmo/nexmo-ruby-code-snippets/pull/44

### CODE REVIEW

**Comment by lornajane** [comment]
@benhayehudi can you take another look at this PR? The build fails because the verify_pages job fails, are we just missing a submodule reference or something?

**Comment by benhayehudi** [comment]
> @benhayehudi can you take another look at this PR? The build fails because the verify_pages job fails, are we just missing a submodule reference or something?

Yes, the submodule PR is awaiting another look by @mheap for it to be merged, hopefully.

File: _examples/account/secret-management/list-all-secrets/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Ruby
+language: ruby
+code:
+    source: .repos/nexmo/nexmo-ruby-code-snippets/account/secret-management/list-all-secrets.rb
+    from_line: 15
```
Comment (lornajane) [inline _examples/account/secret-management/list-all-secrets/ruby.yml]: Can you check these line numbers? I'm looking at https://nexmo-developer-pr-1805.herokuapp.com/account/code-snippets/list-all-secrets and it doesn't seem right to me

Comment (benhayehudi) [reply _examples/account/secret-management/list-all-secrets/ruby.yml]: Thank you, @lornajane. I've updated the line references, you were correct. I had made some changes in the code but had not updated the line references here. Should be good now!

**Pull request merged**